lms/djangoapps/grades/rest_api/v1/gradebook_views.py

```diff
@@ -681,11 +681,7 @@ def _get_user_count(self, query_args, cache_time=3600, annotations=None):
             queryset = queryset.annotate(**annotations)
         queryset = queryset.filter(*query_args)
 
-        cache_key = 'usercount.%s' % queryset.query
-        user_count = cache.get(cache_key, None)
-        if user_count is None:
-            user_count = queryset.count()
-            cache.set(cache_key, user_count, cache_time)
```

Comment:
Looks like you can remove the import for the `cache` now

Like it was done in the [PR to master branch](https://github.com/openedx/edx-platform/pull/33617/files#diff-cb4e621e463ad8ef688692509b83d217efa0cc02682395149a25b37418ddef85L12)